### PR TITLE
Push to rubygems from circleci

### PR DESCRIPTION
**Pull request by bliof**
Push to rubygems when a tag `v1.2.3` is pushed

### CODE REVIEW

**Comment by bliof** [comment]
But.. how are we going to hide the secret 🤕

**Comment by bliof** [comment]
On the https://circleci.com/gh/FundingCircle/workflows/hellgrid I can click some `Rerun from beginning` button and it "flashes" happily 🤷‍♂️

**Comment by bliof** [comment]
Ahh my bad I though I can ssh from everywhere when I do `Rebuild with ssh` in circleci but uses my pub key in github for authentication.

File: .circleci/config.yml [hunk 1]
```diff
@@ -38,29 +46,24 @@ jobs:
...
-            - vendor/bundle
+      - *restore_gems
+      - *bundle_install
+      - *cache_gems
+      - run:
+          name: Build the gem
+          command: gem build $CIRCLE_PROJECT_REPONAME.gemspec
       - deploy:
-          name: Publish to RubyGems
+          name: Publish to rubygems
```
Comment (sgerrand) [inline .circleci/config.yml]: 🙋 This didn't need to change.

File: .circleci/config.yml [hunk 2]
```diff
@@ -72,11 +75,12 @@ workflows:
...
               only: /.*/
-      - deploy:
+      - push-to-rubygems:
+          context: org-global
           requires:
             - test
           filters:
             tags:
-              only: /.*/
+              only: /^v\d.+/
```
Comment (sgerrand) [inline .circleci/config.yml:84]: 👏

**Review by sgerrand — APPROVED** [review]
LGTM, no blockers. 👍

**Pull request merged**